### PR TITLE
Fix broken tests on master and enable Jenkins to run the tests

### PR DESCRIPTION
**Pull request by mcgoooo**
While working on contacts admin, it turned out that the tests are not running on jenkins :man_facepalming: and there was some broken tests, this re enables jenkins running the tests and fixes the broken tests

example of tests not running
https://ci.integration.publishing.service.gov.uk/job/contacts/job/change-rendering-app/3/
with this branch
https://ci.integration.publishing.service.gov.uk/job/contacts/job/fix-broken-tests-on-master/1/

https://trello.com/c/pIBrqrZX

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -61,6 +61,7 @@ node {
 
     stage('Tests') {
       govuk.runRakeTask('ci:setup:rspec spec')
+      govuk.runRakeTask("default")
```
Comment (36degrees) [inline Jenkinsfile]: Might be more explicit to use `govuk.runTests()`

File: Jenkinsfile [hunk 2]
```diff
@@ -61,6 +61,7 @@ node {
 
     stage('Tests') {
       govuk.runRakeTask('ci:setup:rspec spec')
```
Comment (tijmenb) [inline Jenkinsfile]: What does `rake spec` do, if not run the tests?

**Comment by tijmenb** [comment]
Got it. 

We're not setting `RAILS_ENV=test` on CI, so `rake spec` will do nothing. Somehow just running `rake` _will_ run the tests.

So @36degrees's suggestion of using `govuk.runTests()` instead of `govuk.runRakeTask('ci:setup:rspec spec')` will fix this.

File: Jenkinsfile (hunk 2 above)
Comment (mcgoooo) [inline Jenkinsfile]: doesn't do anything locally, not too sure why

**Pull request merged**